ngraph/core/src/op/greater_eq.cpp

```diff
@@ -88,13 +70,6 @@ namespace
     }
 }
 
-bool op::v0::GreaterEq::evaluate(const HostTensorVector& outputs,
```

Comment:
should we rename the file? or it will break binary compatibility and doesn't worth it?

Reply:
We have offline discussion with @ilyachur. File renaming can't provide backward compatibility

Reply:
In general, we can rename this file, but for we also should create a file with name `greater_eq.hpp` which will include the new file.
And after some releases need to remove this `greater_eq.hpp` which is needed only for backward compatibility.